packages/api-reference/src/features/ExampleRequest/ExampleRequest.vue

```diff
@@ -120,40 +108,114 @@ const hasMultipleExamples = computed<boolean>(
     ).length > 1,
 )
 
-const getGlobalSecurity = inject(GLOBAL_SECURITY_SYMBOL)
+/** Current request */
+const request = computed(() => {
```

Comment:
we can probably do a better job with this, but it was my first time coming across harRequest.

Reply:
I think once we do the content-type revamp we can fix this right up